hamster_gtk/hamster_gtk.py

```diff
@@ -363,11 +373,15 @@ def get_db_config():
                 })
             return result
 
+        def get_autocomplete_activities_offset():
+            return cp_instance.getint('Frontend', 'autocomplete_activities_offset')
```

Comment:
Yes. This is somewhat expected / desired. Right now all config keys are mandatory. so a preexisting config without all relevant keys is invalid. As we are not API stable at this point its acceptable that we crash with "invalid" older configs.

Before a major release we probably should provide default fallbacks for non supplied config keys though.